web/src/views/FileBrowserView/PublishFileBrowser.vue

```diff
@@ -194,6 +230,34 @@ export default {
         this.location = `${this.location}${name}`;
       }
     },
+
+    async refreshItems() {
+      const { version, identifier, location } = this;
+
+      this.loading = true;
```

Comment:
I just learned that `asyncComputed` gives you this feature for free: https://github.com/foxbenjaminfox/vue-async-computed#computation-status

Reply:
See 289ac4fc0c6e7df8e5d84449006751099181f084.